### DISCUSSION TITLE
integrazione telecamere con enable/disable sensore movimento

### DISCUSSION THREAD

**Discussion by redaste** [indomus/forum#298] [HUB: Home Assistant]
Ciao, sto valutando la soluzione migliore per integrare telecamere poe hikvision (queste DS-2CD2146G2-I) di futuro acquisto affichè posso essere in grado di fare ciò da home assistant:
- abilitare / disabilitare la rilevazione del movimento nella telecamera (mandatorio) 
- leggere lo stato del motion sensor per triggerare eventi (mandatorio)
- streaming delle camere on demand (nice to have, non obbligatorio)

Quale integrazione mi suggerite per soddisfare le mie necessità?

**Comment by indomus** [comment]
Decisamente questa https://www.home-assistant.io/integrations/hikvision/ devi solo capire se quel modello è pienamente supportato (il sospetto cmq è che lo sia)

**Reply by redaste** [reply]
OK, grazie per la risposta. Questa integrazione copre sicuramente il secondo punto per esporre i sensori, mi rimane il dubbio se è in grado di abilitare/disabilitare il motion sensor
